docs/components/NavLink.js

```diff
@@ -19,7 +19,7 @@ const NavLink = ({ children, ...props }) => {
 };
 
 export const stringToUrl = (str, path = "/") => {
-  return `${path}${str.toLowerCase().split(" ").join("-")}`;
+  return `${path}${str.toLowerCase().split(" ").join("-")}/`;
```

Comment:
can str be null ?

Reply:
str cannot be null.
`const components = ["MenuBar", "Combobox", "MultiSelect"];`
this goes to the str

Reply:
Hope above comment clarified your doubts and the conversation can be resolved?